mirgecom/initializers.py

```diff
@@ -147,6 +147,8 @@ def __call__(self, x_vec, *, t=0, eos=IdealSingleGas()):
         eos: mirgecom.eos.IdealSingleGas
             Equation of state class to supply method for gas *gamma*.
         """
+        if eos is None:
+            eos = IdealSingleGas()
```

Comment:
In what now seems like distant past, someone once insisted that this is a bad idea. He was right.  I think we should eliminate defaulting the EOS - which was done for testing convenience.